apps/frontend/src/components/WorkoutMenu/WorkoutEditor.tsx

```diff
@@ -77,16 +65,21 @@ export const WorkoutEditor = ({
   const { user, saveLocalWorkout } = useUser();
   const toast = useToast();
   const token = (user.loggedIn && user.token) || null;
+  const [showDeleted, setShowDeleted] = React.useState(false);
 
   const canSaveLocally =
     loadedWorkout.type === 'new' || loadedWorkout.type === 'local';
 
   const canSaveRemotely =
     token && (loadedWorkout.type === 'new' || loadedWorkout.type === 'remote');
 
+  const canBeDeleted = token && loadedWorkout.type === 'remote';
+
+  const cancelRef = React.useRef<HTMLButtonElement>(null);
```

Comment:
Have no real idea here, but seems to work.
I guess we could remove this by having own component

Reply:
Should we support deletion of local workouts as well? 🤠

Reply:
Could be done in a later PR ofc 🚀